pyproject.toml

```diff
@@ -119,6 +92,7 @@ lint.ignore = [
     "ISC001",  # Ruff formatter incompatible
     "A002",    # Argument id shadowing scope id
     "ERA001",  # Commented out code
+    "G004",    # flake8-logging-format - logging-f-string
```

Comment:
I think we should not log f-strings 😂 . Something an old colleague drilled into me.